site2/docs/reference-pulsar-admin.md

```diff
@@ -1852,6 +1852,9 @@ Subcommands
 * `get-inactive-topic-policies`
 * `set-inactive-topic-policies`
 * `remove-inactive-topic-policies`
+* `set-max-subscriptions`
+* `get-max-subscriptions`
+* `remove-max-subscriptions`
```

Comment:
Need to add them to [pulsar-admin website](http://pulsar.apache.org/tools/pulsar-admin/2.8.0-SNAPSHOT/#-em-set-max-subscriptions-per-topic-em-)? or is `set-max-subscriptions` same to `set-max-subscriptions-per-topic`? Same comments for the other two commands.

Reply:
`set-max-subscriptions` is the same to `set-max-subscriptions-per-topic`, fixed in PR #9750